src/api/liskApi.js

```diff
@@ -12,212 +12,179 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import { LIVE_PORT, TEST_PORT, GET, POST } from 'constants';
+import {
+	LIVE_PORT,
+	SSL_PORT,
+	TEST_PORT,
+	TESTNET_NETHASH,
+	MAINNET_NETHASH,
+} from 'constants';
 import config from '../../config.json';
-import * as privateApi from './privateApi';
-import * as utils from './utils';
+import * as resources from './resources';
+
+const commonHeaders = {
+	'Content-Type': 'application/json',
+	os: 'lisk-js-api',
+	version: '1.0.0',
+	minVersion: '>=0.5.0',
+};
 
 export default class LiskAPI {
 	constructor(providedOptions) {
 		const options = Object.assign({}, config.options, providedOptions);
+		this.ssl = options.ssl !== false;
+		this.testnet = options.testnet === true;
```

Comment:
Why usually use ` x !== false` before and `x === true` here as a check?

Reply:
`ssl` default is true and `testnet` default is false